src/Type/Definition/FieldDefinition.php

```diff
@@ -179,7 +181,10 @@ public function getArg($name)
      */
     public function getType() : Type
     {
-        return $this->type;
+        $type = Schema::resolveType($this->type);
+        assert($type instanceof OutputType, new Error('Expected type of FieldDefinition to implement OutputType but got: ' . Utils::printSafe($type)));
```

Comment:
I would suggest using `InvariantViolation` instead of an `Error` exception for assertions.

Reply:
Fixed in 932f5dfa75d76125adb8de3609132e3d99b85ebe.

(sorry for the slow progress--I moved to a new computer this week, and work has been hectic)